ui/permissions.py

```diff
@@ -83,6 +90,15 @@ def has_permission(self, request, view):
         return request.user and request.user.is_superuser
 
 
+class IsAdminOrStaff(BasePermission):
+    """IsAdminOrStaff permission to create new collections"""
+    def has_permission(self, request, view):
+        if request.method == 'POST':
+            if not has_staff_permission(request.user):
+                return False
+        return True
```

Comment:
great idea, though the above will need a little tweaking since superusers can create collections for other users but staff can't.